packages/SystemUI/res/values-zh-rCN/yaap_strings.xml

```diff
@@ -0,0 +1,108 @@
+    <string name="quick_settings_sound_label">声音</string>
+    <string name="quick_settings_sound_ring">响铃</string>
+    <string name="quick_settings_sound_vibrate">震动</string>
+    <string name="quick_settings_sound_silent">静音</string>
+
+    <!-- Heads Up QS Tile -->
+    <string name="quick_settings_heads_up_label">弹出通知</string>
+    <string name="accessibility_quick_settings_heads_up_off">弹出通知 关</string>
+    <string name="accessibility_quick_settings_heads_up_on">弹出通知 开</string>
+    <string name="accessibility_quick_settings_heads_up_changed_off">弹出通知 已关闭</string>
+    <string name="accessibility_quick_settings_heads_up_changed_on">弹出通知 已打开</string>
+
+    <!-- USB tethering QS tile -->
+    <string name="quick_settings_usb_tether_label">USB网络共享</string>
+
+    <!-- Gaming Mode -->
+    <string name="gaming_mode_tile_title">游戏模式</string>
+    <string name="gaming_mode_tile_toast">游戏模式已开启</string>
+    <string name="gaming_mode_tile_toast_disabled">游戏模式已关闭</string>
+    <string name="gaming_mode_dialog_title">游戏模式</string>
+    <string name="gaming_mode_dialog_message">游戏模式可以切换弹出通知，硬件按钮，免打扰，导航栏，自动亮度和设置媒体音量的设置值。 退出游戏模式后，所有切换的设置都会被切换回原样。 长按进行设置</string>
+    <string name="accessibility_quick_settings_gaming_mode_on">游戏模式 开</string>
+    <string name="accessibility_quick_settings_gaming_mode_off">游戏模式 关</string>
+    <string name="gaming_mode_panel_headsup_title">禁用弹出通知</string>
+    <string name="gaming_mode_panel_zen_title">启用免打扰</string>
+    <string name="gaming_mode_panel_navbar_title">禁用导航栏</string>
+    <string name="gaming_mode_panel_hw_title">禁用硬件导航键</string>
+    <string name="gaming_mode_panel_brightness_title">禁用自动亮度</string>
+    <string name="gaming_mode_panel_media_title">设置媒体音量</string>
+
+    <!-- AOD QS tile states -->
+    <string name="quick_settings_aod_label">息屏显示</string>
+    <string name="quick_settings_aod_on_charge_label">充电时显示息屏 </string>
+    <string name="quick_settings_aod_off_label">息屏显示 关</string>
+    <string name="quick_settings_aod_off_powersave_label">息屏显示 关\n省电</string>
+
+    <!-- QS Config -->
+    <string name="qs_menu_item_columns">列数</string>
+    <string name="qs_menu_item_columns_landscape">列数（横屏）</string>
+    <string name="qs_menu_item_qs_rows">最大列数</string>
+    <string name="qs_menu_item_columns_two">2</string>
+    <string name="qs_menu_item_columns_three">3</string>
+    <string name="qs_menu_item_columns_four">4</string>
+    <string name="qs_menu_item_columns_five">5</string>
+    <string name="qs_menu_item_columns_six">6</string>
+    <string name="qs_menu_item_columns_seven">7</string>
+    <string name="qs_menu_item_columns_eight">8</string>
+    <string name="qs_menu_item_columns_nine">9</string>
+    <string name="qs_menu_item_reset">重置</string>
+    <string name="qs_menu_item_titles">显示磁贴标题</string>
+    <string name="qs_menu_item_qs_columns">快速栏磁贴个数</string>
+    <string name="qs_menu_item_columns_auto">自动</string>
+    <string name="qs_menu_item_titles">页脚文字</string>
```

Comment:
Which one of `qs_menu_item_titles` is right. We'll never know. I am removing the last one

Reply:
Perhaps Google translate it?

Reply:
yes，Did not find the same，You can delete the last line